### ISSUE TITLE
Loading custom TFLite models via BaseImagePredictor is slow on Android platform

### ISSUE THREAD

**Issue by KadirTilki** [asus4/tf-lite-unity-sample#171]
**Environment:**
 - OS/OS Version: Android 9/Android 11
 - Source Version: master/v2.4.0
 - Unity Version: Unity 2020.3.12.f1

**Describe the bug**
Loading custom TFLite models via BaseImagePredictor is very slow on Android platform compared to other platforms. I have timed each operation and found out that `interpreter = TfLiteInterpreterCreate(model, options.nativePtr);` call inside the Interpreter class is the cause. On Android devices (Pixel 4a, Samsung Galaxy Tab S4 & S6) that operation takes around 900 ms to complete and on an iPad Pro 2nd generation it only takes around 32 ms. 

**To Reproduce**
1. Create an image predictor with a custom TFLite model (e.g., https://tfhub.dev/tensorflow/lite-model/mobilenet_v1_1.0_224/1/default/1)
2. Deploy on an Android and an iOS device.
3. Measure the time to create the image predictor on both devices.
4. Compare the measurements.

**Expected behavior**
Creating an image predictor with a custom TFLite model on Android platform should not be very slow compared to other platforms.

**Comment by asus4** [comment]
Does it take the same amount of time in native TensorFlow Lite? If so, please open an issue to [TensorFlow's GitHub](https://github.com/tensorflow/tensorflow/).

If we change something on the Unity side, some Android are very slow to load files from SD cards, so making the model load asynchronously might improve it.

**Comment by KadirTilki** [comment]
Hello, thank you for the quick response. I tested a few custom tflite models on [the TensorFlow example project](https://github.com/tensorflow/examples/tree/master/lite/examples/image_classification/android) and model loading took 25 ms at most.

Regarding your other point, the devices I tested with do not have an SD card. Also I have tried loading the models from different folders (Resources, StreamingAssets, Application.persistentDataPath) and there was not a visible time difference loading from these folders and all of them took around 900 ms.
